configuration/analyzer_config.json

```diff
@@ -2992,6 +2992,23 @@
     "secrets": {},
     "params": {}
   },
+  "OnionScan": {
+    "type": "observable",
+    "python_module": "onionscan.OnionScan",
+    "description": "check a Tor .onion domain for privacy misconfigurations and info leaks.",
+    "disabled": false,
+    "external_service": false,
+    "leaks_info": false,
+    "observable_supported": [
+      "domain"
+    ],
+    "config": {
+      "soft_time_limit": 30,
+      "queue": "default"
+    },
+    "secrets": {},
+    "params": {}
```

Comment:
should add support for 2 parameters:

- `verbose`. type: `bool`, default: `True`.
- `torProxyAddress`. type: `str`, default: `None`.

Reply:
the `command` should be modified on the basis of these parameter values. Refer here: https://github.com/s-rah/onionscan#quick-start

Reply:
Ok, Sir, I've added `--verbose` and `--torProxyAddress` support as parameters.